tests/admin_docs/tests.py

```diff
@@ -103,6 +116,31 @@ def test_view_detail_illegal_import(self):
         self.assertEqual(response.status_code, 404)
         self.assertNotIn("urlpatterns_reverse.nonimported_module", sys.modules)
 
+    @unittest.skipIf(six.PY2, "Python 2 doesn't support __qualname__.")
+    def test_view_detail_as_method(self):
+        """
+        #27018 - Ensure views that are methods can be displayed
+        """
+        url = reverse(
```

Comment:
A single line looks okay to me (did you read in our style guide how we allow lines up to 119 characters?).